.gitignore

```diff
@@ -11,6 +11,7 @@ schemastore
 .venv*
 # Formatter debugging (crates/ruff_python_formatter/README.md)
 scratch.py
+scratch.pyi
```

Comment:
Should it be `scratch.*`. 

You can also use Jetbrains scratch buffer. They get stored outside of the project.

Reply:
i use scratch files a lot but they don't work well with external commands (they don't have a stable location)